mrchecker-framework-modules/mrchecker-core-module/src/main/java/com/capgemini/mrchecker/test/core/ModuleType.java

```diff
@@ -1,6 +1,7 @@
 package com.capgemini.mrchecker.test.core;
 
 public enum ModuleType {
+	CORE,
```

Comment:
From what i know core is a module that connect all sub modules and should not be present here.

Reply:
I need that to unit test CORE classes.